server/interface.go

```diff
@@ -8,9 +8,10 @@ import (
 
 //Client interface for mysterium centralized api - will be removed in the future
 type Client interface {
-	RegisterIdentity(identity identity.Identity, signer identity.Signer) (err error)
-	NodeRegister(proposal dto_discovery.ServiceProposal) (err error)
-	NodeSendStats(nodeKey string) (err error)
 	FindProposals(nodeKey string) (proposals []dto_discovery.ServiceProposal, err error)
+	//these functions have side effects - therefore signer required
```

Comment:
Signer is not required only because these functions have side-effects:
In the future we might have `GET` endpoints without side-effects, which requires authorization.
What about this?
```
//these functions are signed because they require authorization
```

Reply:
Looks better. Fixed